services/QuillLMS/app/helpers/footer_helper.rb

```diff
@@ -64,4 +64,11 @@ def about_us_links
       { href: '/contact', label: 'Contact Us' }
     ]
   end
+
+  def show_footer(path)
+    excluded_paths = ['/teachers/premium_hub/usage_snapshot_report', '/teachers/premium_hub/data_export']
+    return false if excluded_paths.include?(path)
+
+    true
```

Comment:
Since this method is return a boolean lets rename it `show_footer?`.  You could also pull out a constant here:

```
 EXCLUDED_FOOTER_PATHS = [
  '/teachers/premium_hub/usage_snapshot_report', 
  '/teachers/premium_hub/data_export'
].freeze

def show_footer?(path)
  !path.in?(EXCLUDED_FOOTER_PATHS)
end
```